addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -398,6 +398,12 @@ public function index($category = null, $city = null)
             $this->template->set('meta_title', $user->name() . ' ' . trans('visiosoft.module.advs::field.ads'));
         }
 
+        // Set rel="canonical"
+        $canonParam = $param;
+        unset($canonParam['sort_by'], $canonParam['doping']);
+        $canonUrl = fullLink($canonParam, \request()->url());
+        $this->template->set('additional_meta', "<link rel='canonical' href='$canonUrl'/>");
```

Comment:
it's better to use it if there is sort_by in link. Otherwise it will not all right